modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1456,6 +1456,13 @@ public void testAddObjectEntryWithObjectValidationRule() throws Exception {
 		DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(
 			"yyyy-MM-dd HH:mm");
 
+		LocalDateTime todayLocalDateTime = LocalDateTime.now();
+
+		LocalDateTime tomorrowLocalDateTime = todayLocalDateTime.plusDays(1);
+
+		String localDateTimeString = dateTimeFormatter.format(
+			tomorrowLocalDateTime);
```

Comment:
Hey @diogomelo09 rename  `localDateTimeString` to `tomorrowLocalDateTimeString`.

Thanks!

Reply:
Done!! :white_check_mark: